SECURITY.md

```diff
@@ -0,0 +1,20 @@
+# Security Policy
+
+## Supported Versions
+
+See releases section for versions of Alby that are currently being supported with 
+security updates: https://github.com/getAlby/lightning-browser-extension/tags
+
+## Reporting a Vulnerability
+
+If you found a security vulnerability on this platform please send an email to hello@michaelbumann.com (do not use this email for asking any kind of support).
```

Comment:
```suggestion
If you found a security vulnerability on this platform please send an email to security@getalby.com (do not use this email for asking any kind of support).
```

Reply:
shall I remove the PGP fingerprint then? it's for hello@michaelbumann.com

Reply:
resolved